include/fastdds/rtps/transport/UDPv6TransportDescriptor.h

```diff
@@ -35,13 +35,16 @@ class TransportInterface;
  */
 typedef struct UDPv6TransportDescriptor : public UDPTransportDescriptor
```

Comment:
Avoid typedef trick.
Check class documentation, as it seems to only mention fields on SocketTransportDescriptor